### DISCUSSION TITLE
Configuring component styles using `extendTheme`

### DISCUSSION THREAD

**Discussion by jithinlal** [chakra-ui/chakra-ui#5538] [Q&A]
So I want to configure my own components. Like Button, Input. But the component structure is so difficult to find in the documentation. 
Like when I want to change the borderColor of input component, it should be written like it's mentioned in [this](https://github.com/chakra-ui/chakra-ui/issues/2347#issuecomment-878629255) link. Is there any specific documentation that I miss, a one place stop where I could see the supported Components to override?

**Comment by old-lhpegnolatto** [comment]
Hi @jithinlal!

If there is documentation for this, I don't know either.

But I suggest you look directly at the [default ChakraUI theme](https://github.com/chakra-ui/chakra-ui/tree/main/packages/theme/src), it will definitely help you understand how to configure components and foundations.
